app/views/transcribe/translate.html.slim

```diff
@@ -68,7 +68,8 @@
       -if @preview_xml
         .page-preview ==xml_to_html(@preview_xml)
 
-      .page-editarea(style=("display:none" if @preview_xml))
+      -direction = Rtl.rtl?(@collection.text_language) ? 'rtl' : 'ltr'
+      .page-editarea(style=("display:none" if @preview_xml) lang="#{@collection.text_language}" dir=direction class=direction)
```

Comment:
On the assumption that most translation target languages continue to be LTR (mostly English, but likely Spanish soon), we should deconditionalize this.  When we need to support RTL target languages for translation, we'll need to create a new `target_language` field on `collections` and drive the logic off of that, rather than the source text.